sources/north-america/us/fl/Miami-Dade_Ortho_2021.geojson

```diff
@@ -0,0 +1,266 @@
+{
+    "type": "Feature",
+    "properties": {
+        "id": "Miami-Dade_Ortho_2021",
+        "attribution": {
+            "url": "https://gis-mdc.opendata.arcgis.com/pages/imagery",
+            "text": "Miami-Dade County",
+            "required": false
+        },
+        "name": "Miami-Dade County Orthoimagery (2021)",
+        "icon": "https://www.miamidade.gov/resources/global/images/md-logo-color.png",
+        "url": "https://imageserverintra.miamidade.gov/arcgis/services/Woolpert2021/ImageServer/WMSServer?format=image/jpeg&version=1.3.0&service=wms&request=getmap&layers=0&styles=&crs={proj}&width={width}&height={height}&bbox={bbox}",
+        "min_zoom": 5,
+        "max_zoom": 19,
+        "license_url": "https://wiki.openstreetmap.org/wiki/Florida#Publicly_Available_Data",
+        "permission_osm": "explicit",
+        "country_code": "US",
+        "type": "wms",
+        "available_projections": [
+            "CRS:84",
+            "EPSG:4326",
+            "EPSG:6438"
+        ],
+        "start_date": "2021",
+        "end_date": "2021",
+        "description": "The 2021 orthoimagery for Miami-Dade County of the State of Florida",
+        "category": "historicphoto",
```

Comment:
is this 2021 imagery historical?

Reply:
My thought was because they have a most recent layer, all other layers of specific year should be considered as historical. The county has adequate funding which can be used to update aerial every year.

Correct me if I’m wrong.

Reply:
I can't see any other us/fl imagery already in the index, which is the more recent layer?

Reply:
It’s in this PR…

Reply:
ah I see, thanks.